tendrl/node_agent/message/logger.py

```diff
@@ -34,6 +34,11 @@ def __init__(self, message):
     def push_operation(self):
         NS.etcd_orm.client.write(
             "/Messages/jobs/%s" % self.message.job_id,
+            '',
```

Comment:
Why are you passing an empty string as value? It is not required https://github.com/jplana/python-etcd/blob/0.4.5/src/etcd/client.py#L482